tests/manual-test-cases/Group19-ROBO/19-1-ROBO-SKU.md

```diff
@@ -15,10 +15,10 @@ This test requires access to VMware Nimbus cluster for dynamic ESXi and vCenter
 2. Add the ROBO SKU license to the vCenter appliance
```

Comment:
I believe the environment of primary interest has an Enterprise license for VC but ROBO for the hosts.

Reply:
Thanks - I'm changing this line to `Add the Enterprise license to the vCenter appliance`. cc @mhagen-vmware since he wrote this particular test.